flocker/volume/filesystems/memory.py

```diff
@@ -177,8 +178,19 @@ def destroy(self, volume):
     def set_maximum_size(self, volume):
         filesystem = self.get(volume)
         root = filesystem.get_path()
+
+        def get_size():
+            total_size = 0
+            for f in root.walk():
+                if not f.isdir():
+                    total_size += f.getsize()
+            return total_size
+
+        current_size = get_size()
         size_path = root.child(b".size")
         if volume.size.maximum_size is not None:
+            if volume.size.maximum_size < current_size:
+                raise MaximumSizeTooSmall
```

Comment:
Hm.  I wish the style checker pointed this out because it probably is just a matter of style - but typically we raise instances, not types (ie, `raise MaximumSizeTooSmall()` instead).  The fact that both of these are allowed is a weird quirk resulting from an internal CPython implementation detail.

But as I said it is basically just a style thing.  I can't think of any behavior that differs based on which you use.  And since it's apparently not part of the style the pep8 tool checks for it's not really enforced.